### PR TITLE
Add support for PsySH

### PR DESCRIPTION
**Pull request by nesk**
[PsySH](https://psysh.org/) is a popular PHP REPL.

### CODE REVIEW

File: mackup/applications/psysh.cfg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[application]
+name = PsySH
+
+[configuration_files]
```
Comment (lra) [inline mackup/applications/psysh.cfg:4]: This should be an `xdg_configuration_files`: https://github.com/lra/mackup/tree/master/doc#add-support-for-an-application-using-the-xdg-directory

**Review by lra — COMMENTED** [review]
Can you add a line in the README and the CHANGELOG? See the "contributing" link when you create a PR.

**Comment by nesk** [comment]
Sorry, I went too fast to submit this PR. Everything should be fixed. 🙂

**Review by lra — APPROVED** [review]

**Comment by lra** [comment]
thanks

**Pull request merged**